### ISSUE TITLE
Pull in data and info for upcoming and current shutdowns

### ISSUE THREAD

**Issue by devinmatte** [transitmatters/shutdown-tracker#27]
https://www.mbta.com/alerts/subway?alerts_timeframe=upcoming

Use the API to be a source of info

**Comment by DaAwesomeP** [comment]
Current shutdowns should be available as GTFS alerts. I'm not sure if GTFS shows upcoming shutdowns.

**Comment by devinmatte** [comment]
Yeah I started looking into things and it only helps us keep Current shutdowns up to date, and via the alerts, upcoming in the next 7 days. Would be good to catch last minute changes to shutdowns
